claims_p1_test.go

```diff
@@ -0,0 +1,237 @@
+// Copyright 2021-2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package psatoken
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func mustBuildValidP1Claims(t *testing.T, includeOptional, noSwMeasurements bool) IClaims {
+	c, err := newP1Claims(includeOptional)
```

Comment:
NewP1Claims take includeProfile boolean while here includeOptional is passed ?

Reply:
yep